pystiche/image/_transforms.py

```diff
@@ -0,0 +1,129 @@
+"""kornia compatibility layer
+
+This module exists only to provide compatibility to improve kornia functionality until
+it is fixed upstream. The following improvements are included:
+
+- The default value of 'align_corners' (resize, Affine) now works by default with every
+  interpolation mode
+- Tensor attributes of Transformations (Affine, Normalize, Denormalize) are now
+  registered as non-persistent buffers by default.
+- Parameters that should be passed as tensors (Affine, Normalize, Denormalize) are now
+  allowed as float or tuple of floats for convenience
+- The Affine transformation now is independent of the batch_size
+"""
```

Comment:
Unfortunately, as of `kornia==0.4.1` we need a compatibility layer, since `kornia` is not able to handle our use case completely.